lib-multisrc/mayotune/src/eu/kanade/tachiyomi/multisrc/mayotune/MayoTune.kt

```diff
@@ -0,0 +1,162 @@
+    override fun popularMangaFromElement(element: Element) = throw UnsupportedOperationException()
+
+    // Latest
+    override val supportsLatest: Boolean = true
+    override fun fetchLatestUpdates(page: Int): Observable<MangasPage> {
+        return Observable.just(MangasPage(sourceList, false))
+    }
+
+    override fun latestUpdatesRequest(page: Int): Request = throw UnsupportedOperationException()
+    override fun latestUpdatesNextPageSelector(): String? = throw UnsupportedOperationException()
+    override fun latestUpdatesSelector(): String = throw UnsupportedOperationException()
+    override fun latestUpdatesFromElement(element: Element): SManga =
+        throw UnsupportedOperationException()
+
+    // Search
+    override fun fetchSearchManga(
+        page: Int,
+        query: String,
+        filters: FilterList,
+    ): Observable<MangasPage> {
+        val mangas = mutableListOf<SManga>()
+        sourceList.map {
+            if (it.title.lowercase().contains(query.lowercase())) {
+                mangas.add(it)
+            }
+        }
+        return Observable.just(MangasPage(mangas, false))
+    }
+
+    override fun searchMangaRequest(page: Int, query: String, filters: FilterList) =
+        throw UnsupportedOperationException()
+
+    override fun searchMangaNextPageSelector() = throw UnsupportedOperationException()
+    override fun searchMangaSelector() = throw UnsupportedOperationException()
+    override fun searchMangaFromElement(element: Element) = throw UnsupportedOperationException()
+
+    // Get Override
+    override fun mangaDetailsRequest(manga: SManga): Request {
+        return GET(manga.url, headers)
+    }
+
+    override fun chapterListRequest(manga: SManga): Request {
+        return GET(manga.url + "api/chapters", headers)
+    }
+
+    override fun pageListRequest(chapter: SChapter): Request {
+        return GET(chapter.url, headers)
+    }
+
+    // Details
+
+    override fun mangaDetailsParse(document: Document): SManga = SManga.create().apply {
+        val statusTxt = document.select("div.text-center").find {
+            it.text().contains("Status")
+        }?.text()?.substringBefore("Status")?.trim()
+
+        url = sourceList.first().url
+        title = sourceList.first().title
+        artist = sourceList.first().artist
+        author = sourceList.first().author
+        description = document.select(".text-lg").text()
+        genre = document.select("span.text-sm:nth-child(2)").text().replace("•", ",")
+        status = when (statusTxt) {
+            "Ongoing" -> SManga.ONGOING
+            "Completed" -> SManga.COMPLETED
+            "Cancelled" -> SManga.CANCELLED
+            "Hiatus" -> SManga.ON_HIATUS
+            "Finished" -> SManga.PUBLISHING_FINISHED
+            else -> SManga.UNKNOWN
+        }
+        thumbnail_url = if (document.select(".object-contain").attr("src").isNotEmpty()) {
+            baseUrl + document.select(".object-contain").attr("src")
```

Comment:
Use absolute URLs for this
```suggestion
        thumbnail_url = if (document.select(".object-contain").absUrl("src").isNotEmpty()) {
            baseUrl + document.select(".object-contain").absUrl("src")
```